Gemfile.lock

```diff
@@ -418,4 +422,4 @@ DEPENDENCIES
   whenever (= 0.9.4)
 
 BUNDLED WITH
-   1.11.2
+   1.12.5
```

Comment:
I'd expect it's still 1.11.2 actually - that's what it is on integration. I'll fix it. 

Does this cause more pain than just a warning? I reckon I just updated my bundler at some point for a different repo.